Framework/PythonInterface/mantid/simpleapi.py

```diff
@@ -266,6 +266,10 @@ def handleSpecialProperty(name, value=None):
         try:
             if value is None:
                 value = kwargs.pop(name)
+            else:
+                # We don't need the value, but still need to remove from kwargs
+                # so that this property isn't set again later
+                kwargs.pop(name, None)
```

Comment:
Arguably no need for the `else:`... could just always do this one indent up (which is how it was before!). But it makes no difference either way. The comment definitely helps, I can see how it wasn't clear without it.

No changes needed.